packages/elements/src/components/ino-avatar/ino-avatar.scss

```diff
@@ -122,10 +132,49 @@
   border-radius: 50%;
   box-sizing: border-box;
   position: relative;
+  justify-content: center;
 
   &.initials {
     background: theme.$p-2;
   }
+
+  .loading-wrapper {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+    height: 100%;
+    width: 100%;
+    background: #FFF;
+
+    z-index: 10;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .loader {
+      width: calc(
+        var(--avatar-size) * (48 / 138)
+      ); // loaders width as a proportion of the avatar size
+      height: calc(var(--avatar-size) * (48 / 138));
+      border: calc(var(--avatar-size) * (5 / 138)) solid #fff;
+      border-bottom-color: theme.$primary;
+      border-radius: 50%;
+      display: inline-block;
+      box-sizing: border-box;
+      animation: spinner 1s linear infinite;
+    }
+
+    @keyframes spinner {
+      0% {
+        transform: rotate(0deg);
+      }
+      100% {
+        transform: rotate(360deg);
+      }
```

Comment:
I feel like the animation might be a bit too intrusive. Imagine having a list of employees that's shown and all avatars are fetching at the same time. This might be a little bit too much for the user with our current animation.

What do you think about some skeleton-like animation like [this one](https://codepen.io/maoberlehner/pen/bQGZYB)?

Reply:
Good idea. I added a modified skeleton-animation for the ino-avatar, check out the video in the PR description.